internal/topo/topotest/rule_test.go

```diff
@@ -22,6 +22,207 @@ import (
 	"github.com/lf-edge/ekuiper/pkg/api"
 )
 
+func TestSRFSQL(t *testing.T) {
```

Comment:
Maybe add most tests in the operator level? Here is more expensive.

Reply:
it's too hard to build testcase in operator level for this case.